docs/userGuide/components/diagrams.md

```diff
@@ -0,0 +1,41 @@
+{% set title = "Diagram Components" %}
+{% set filename = "diagrams" %}
```

Comment:
Should we also change the site title and filename to include the `Image` part as well?

Reply:
Updated